Content.Server/CartridgeLoader/Cartridges/CriminalRecordsCartrideSystem.cs

```diff
@@ -0,0 +1,58 @@
+using Content.Shared.CartridgeLoader;
+using Content.Shared.CartridgeLoader.Cartridges;
+using Robust.Shared.Prototypes;
+using Content.Server.StationRecords.Systems;
+using Content.Shared.CriminalRecords;
+using Content.Shared.CriminalRecords;
+using Content.Shared.Security;
+
+namespace Content.Server.CartridgeLoader.Cartridges;
+
+public sealed class CriminalRecordsCartridgeSystem : EntitySystem
+{
+  [Dependency] private readonly StationRecordsSystem _StationRecords = default!;
+  [Dependency] private readyonly StationSystem _StationSystem = default!;
+
+  [ValidateProtoTypeId<EntityPrototype>]
+  private const string CartridgePrototypeName = "CriminalRecordsCartridge";
+
+  public override void Initialize()
+  {
+    base.Initialize();
+    SubscribeLocalEvent<CriminalRecordsCartridgeComponent, CartridgeMessageEvent>(OnUiMessage) 
+    SubscribeLocalEvent<CriminalRecordsCartridgeComponent, CartridgeUiReadyEvent>(OnUiReady); 
+    SubscribeLocalEvent<ProgramInstallationAttempt>(OnInstallationAttempt);
+  }
+
+  private void OnUiMessage(EntityUid uid, CriminalRecordsCartridgeComponent comp, CartridgeMessageEvent args)
```

Comment:
```suggestion
  private void OnUiMessage(Entity<CriminalRecordsCartridgeComponent> cartridge, ref CartridgeMessageEvent args)
```
use new Entity<> template

Reply:
I don't know how to do this properly. could you show me everything I would need to change :>